topostats/tracing/dnatracing.py

```diff
@@ -121,13 +126,28 @@ def trace_dna(self):
         elif len(self.disordered_trace) >= self.min_skeleton_size:
             self.linear_or_circular(self.disordered_trace)
             self.get_ordered_traces()
+            self.get_trace_heights_and_distances()
             self.linear_or_circular(self.ordered_trace)
             self.get_fitted_traces()
             self.get_splined_traces()
             # self.find_curvature()
             # self.saveCurvature()
             self.measure_contour_length()
             self.measure_end_to_end_distance()
+
+            mask = np.zeros(self.image.shape)
+            for coord in self.ordered_trace:
```

Comment:
Shouldn't need this I don't think.

Under `trace_image()` we call `prep_arrays()` and that crops the padded bounding boxes out of the original image and the labelled region. Before returning them though it [flips every labelled region to be `1` instead of its label](https://github.com/AFM-SPM/TopoStats/blob/08f807cc6d0acf5e15c7aabb63491f20bee622be/topostats/tracing/dnatracing.py#L781) and so `self.grain` should be a mask of `0` and `1` that can be passed straight into `Images()`.